### DISCUSSION TITLE
Call to a member function with() on null at HigherOrderTapProxy.php

### DISCUSSION THREAD

**Discussion by Jacotheron** [livewire/livewire#8561] [1. Help]
I am getting the above error on my site, and have been unable to solve it. The error is typically experienced by other users (when I try it, no errors so far), on a Livewire Controller where they need to upload an Excel sheet for importing. Livewire v3.5.0; Laravel 11.10; PHP 8.3

> Call to a member function with() on null {"userId":4,"exception":"[object] (Error(code: 0): Call to a member function with() on null at /home/forge/<sitename>/vendor/laravel/framework/src/Illuminate/Support/HigherOrderTapProxy.php:34)

Partial Stack Trace:

> #0 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/Drawer/Utils.php(154): Illuminate\\Support\\HigherOrderTapProxy->__call()
> #1 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/Mechanisms/HandleComponents/HandleComponents.php(276): Livewire\\Drawer\\Utils::generateBladeView()
> #2 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/Mechanisms/HandleComponents/HandleComponents.php(231): Livewire\\Mechanisms\\HandleComponents\\HandleComponents->getView()
> #3 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/Mechanisms/HandleComponents/HandleComponents.php(104): Livewire\\Mechanisms\\HandleComponents\\HandleComponents->render()
> #4 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/LivewireManager.php(97): Livewire\\Mechanisms\\HandleComponents\\HandleComponents->update()
> #5 /home/forge/1000kmchallenge.co.za/vendor/livewire/livewire/src/Mechanisms/HandleRequests/HandleRequests.php(93): Livewire\\LivewireManager->update()

As can be seen, the error occurs in the Livewire code, and it appears to be looking for the Livewire Controller, but unable to find/load it.

This particular Controller is controlled by LivewireUI\Modal\ModalComponent (so a button which shows the modal, where the user can pick their file to import). The error only occurs when the user has the modal open, and selected the file (which would automatically trigger the import).

Any idea where to look for what is causing this?

**Comment by brandopnw** [comment]
In case this might help someone else...

I just had this exact error after a bit of refactoring.  It turns out I had commented out the "return view(...)" inside the render function of the livewire component.
